### PR TITLE
Add the vbmc ipmitool command tests in functional test.

### PR DESCRIPTION
**Pull request by helloarys**
Change the .functionaltests from mod 644 to mod 755.
@PaynePei @MarkMa001 @fub2 @JuneZhou19 
Pass the local test on my ubuntu 16.04.

### CODE REVIEW

**Comment by sharkconi** [comment]
@turtle-fly @PaynePei @fub2 @JuneZhou19 不敢想象以前的puffer就是这么测试的？

**Comment by turtle-fly** [comment]
According to discussion, this mixed test includes two test actually:
- emu data test 
- ipmi command response test

@helloarys 
The ipmi reponse test is testing if openipmi load the vbmc.conf correctly. This is not coupled with node personality. So in this test, a good practice is start a default node type and test if ipmi command response correctly. You should not start each node type and test same things, which is not value-added and make the test bloated.

The emu data test is another thread which requires a elaborate discussion.

**Comment by helloarys** [comment]
@turtle-fly @MarkMa001 Done

File: test/functional/test_ipmi_command_response.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+class test_ipmicommand_response_test(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        ipmi.ipmi_start()
+
+    @classmethod
+    def tearDownClass(cls):
+        ipmi.ipmi_stop()
+	time.sleep(3)
```
Comment (turtle-fly) [inline test/functional/test_ipmi_command_response.py]: indent mismatch

Comment (PaynePei) [inline test/functional/test_ipmi_command_response.py]: what's this?

**Comment by turtle-fly** [comment]
:+1:

**Comment by sharkconi** [comment]
+1

**Pull request merged**